buttons:
  manual_penalty: True
  speak_active: True
  short_time: 2